### PR TITLE
Fix for Issue #549 : Add Fallbacks in Aggregator Service

### PR DESCRIPTION
**Pull request by jarpit96**
Changes:
Add fall backs in Aggregator Service when other microservices fail. Default error messages (Product Title and Inventory) are returned to the user.

### CODE REVIEW

File: aggregator-microservices/aggregator-service/src/main/java/com/iluwatar/aggregator/microservices/Aggregator.java [hunk 1]
```diff
@@ -50,8 +50,21 @@
...
+    String productTitle = informationClient.getProductTitle();
+    Integer productInventory = inventoryClient.getProductInventories();
+
+    if (productTitle != null) {
+      product.setTitle(productTitle);
+    } else {
+      product.setTitle("Error: Fetching Product Title Failed"); //Fallback to error message
+    }
+
+    if (productInventory != null) {
```
Comment (iluwatar) [inline aggregator-microservices/aggregator-service/src/main/java/com/iluwatar/aggregator/microservices/Aggregator.java]: `ProductInventoryClientImpl` does not return null but 0 when exception occurs. Perhaps returning null would be the correct way, so the change should be implemented in the client..

**Comment by iluwatar** [comment]
Ping @jarpit96, some changes have been requested. Also the branch is conflicting now.

**Comment by jarpit96** [comment]
Completely missed the comments, I will get on it soon. Will look at the conflicts too.

**Comment by iluwatar** [comment]
Thanks @jarpit96, this looks good!

**Pull request merged**